### ISSUE TITLE
my account was deactivated and I don't know why

### ISSUE THREAD

**Issue by Sam-coder183** [hackclub/arcade-constitution#8]
I tried to contact to the hackclub team via arcade@hackclub.com and I haven't received any response, if I violated any rule please tell me.

I can't access the slack workspace for arcade club, I try to access it via the invitation link and it pops up the following message:

![account deactivated](https://github.com/user-attachments/assets/e1e5b760-3e85-4355-9168-d348b953158d)

the text says (in Spanish) It seems like your account in your hackblub workspace has been deactivated.

kind regards.

Samuel Mateo Rodriguez.

**Comment by Sam-coder183** [comment]
I'm working on alpinecade, an embedded distro made for low powered machines, I decided to switch from alpine linux to artix cause I can't figure out some things about the previous one, but maybe I will make a fork for alpine in the future.

You can check out my repository at: [https://github.com/Sam-coder183/AlpineCade](url)

**Comment by polytroper** [comment]
What email did you use to register for slack? I will look into it

**Comment by Sam-coder183** [comment]
samuelmateorodriguez183@gmail.com

**Comment by Sam-coder183** [comment]
Hi, my account in slack is yet to be reactivated. 

I want to keep participating in the arcade contest for motivating me to release my first alpha of the distro that I'm working on (I've decided to move from alpine to devuan because is more user friendly and based on debian and I can use an alternative init system like runit cause I hate systemd) and then I will try to make it multiplatform with automated syncs using google firestone for making a rapid prototype and then find a way so people can self-host their own game sync servers.

I will be pleased to continue to participate in your next hackatons cause In my country (Spain) the maker culture is not as prominent as in other countries like the US and having a community that I can rely on and share my proyects would be awesome for me.

thanks for considering my issue and I will keep track on your new hackatons for making my software and arduino proyects with this new cool community that I've discovered this summer, but for making that I assume that I have to have access to your slack workplace.

kind regards.

Samuel Mateo Rodriguez.

📍 Huesca, Spain.
